src/vaults/mainnet.json

```diff
@@ -405,6 +405,13 @@
       "logoURI": "https://res.cloudinary.com/duv0g402y/image/upload/v1746534876/tokens/default.png",
       "url": "https://goldilocksdao.io",
       "description": "Almost all supply currently in Beradrome farm."
+    },
+    {
+      "name": "SatLayer",
+      "logoURI": "https://res.cloudinary.com/duv0g402y/image/upload/v1746534876/tokens/default.png",
```

Comment:
do you want to provide an image for the protocol logo? 

also, keeping those assets you removed is preferable

Reply:
added back logos
re: protocol logo, yes! sent the image on TG